lighthouse-core/formatters/partials/critical-request-chains.html

```diff
@@ -1,62 +1,3 @@
-<style>
```

Comment:
Can we restore these files and keep the related styles within the partials? That doesn't seem necessary for this PR?